local-patches/gcc/4.8.5/0030-cygwin64.patch

```diff
@@ -0,0 +1,17 @@
+diff -durN a/gcc/config.host b/gcc/config.host
+--- a/gcc/config.host   2016-10-04 18:01:56.180731000 +1100
++++ b/gcc/config.host   2016-10-04 17:50:36.127000000 +1100
+@@ -221,6 +221,13 @@
+     host_exeext=.exe
+     host_lto_plugin_soname=cyglto_plugin-0.dll
+     ;;
++  x86_64-*-cygwin*)
++    host_xm_file=i386/xm-cygwin.h
++    out_host_hook_obj=host-cygwin.o
++    host_xmake_file="${host_xmake_file} i386/x-cygwin"
++    host_exeext=.exe
++    host_lto_plugin_soname=cyglto_plugin-0.dll
++    ;;
```

Comment:
I've taken a look at the [mainline patch](https://github.com/jcmvbkbc/gcc-xtensa/commit/8f3115a2a840e3e450584a30911b18572add1a89) that adds x86_64 cygwin support, it does more than that. Don't we need at least changes to gcc/config.build? There's the whole series that added x86_64 support, why don't we just backport it?